src/report/utility-reports/hello-world.scm

```diff
@@ -354,9 +354,9 @@ or extending existing reports.")))
 new, totally cool report, consult the mailing list %s.")
           (gnc:html-markup-anchor 
            "mailto:gnucash-devel@gnucash.org"
-           (gnc:html-markup-tt "gnucash-devel@gnucash.org")))
-         (_ "For details on subscribing to that list, see &lt;http://www.gnucash.org/&gt;.")
-         (_ "You can learn more about writing scheme at &lt;http://www.scheme.com/tspl2d/&gt;."))
+           (gnc:html-markup-tt "gnucash-devel@gnucash.org"))
+          (_ "For details on subscribing to that list, see &lt;http://www.gnucash.org/&gt;. ")
+          (_ "You can learn more about writing scheme at &lt;http://www.scheme.com/tspl2d/&gt;.")))
```

Comment:
Whoops you're right.

In my Windows 7 machine the original had 4 elements within html-markup-p. It exports to HTML with correct markup, but within gnucash there is overwritten text after the <a href> element. I guess this is just a Webkit bug. 

My patched one is wrong because gnc:html-markup/format only needs 2 elements and will discard the other 2. Fixing bugs is hard!

![image](https://user-images.githubusercontent.com/1975870/27938856-70bdded6-62f3-11e7-915e-7afe864cb1d5.png)